.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
```

Comment:
Can we cite the tutorial we based our approach on?

Reply:
I'm not sure what you are asking for here

Reply:
Usually, instead of reviewing myself, I search and crosscheck against how others have done something. I search and found another tutorial that wasn't quite in sync, so was suggesting a link to whatever tutorial you used (assuming one is used). Generally, I try to put "See: https://example.com" whenever I adapt from somewhere, and was suggesting that practice :)